src/views/CourtOrder.vue

```diff
@@ -0,0 +1,696 @@
+                            <span>Add a Document</span>
+                          </v-btn>
+
+                          <p
+                            v-if="file"
+                            class="ml-3 pt-6"
+                          >
+                            {{ file.name }}
+                          </p>
+                          <p
+                            v-else-if="isCourtOrder && !notation && enableValidation"
+                            class="ml-3 pt-6"
+                          >
+                            {{ courtOrderCustomValidationMsg }}
+                          </p>
+                        </div>
+                      </v-col>
+                    </v-row>
+
+                    <v-row class="mt-4">
+                      <v-col
+                        cols="12"
+                        md="3"
+                      >
+                        <label class="title-label">Court Order Text</label>
+                      </v-col>
+                      <v-col>
+                        <v-textarea
+                          ref="notationFormRef"
+                          v-model="notation"
+                          class="notation-textarea xmt-4"
+                          filled
+                          :label="notationLabel"
+                          :rows="isCourtOrder ? 2: 5"
+                          :no-resize="true"
+                          :rules="enableValidation ? notationRules : []"
+                          :counter="NOTATION_MAX_LENGTH"
+                        />
+                      </v-col>
+                    </v-row>
+                  </v-form>
+                </v-card>
+              </div>
+            </section>
+          </article>
+          <section>
+            <header>
+              <h2>Staff Payment</h2>
+            </header>
+            <div
+              id="staff-payment-section"
+              :class="{ 'invalid-section': !staffPaymentValid && showErrors }"
+            >
+              <v-card
+                flat
+                class="pt-6 px-4"
+              >
+                <StaffPaymentShared
+                  :staffPaymentData.sync="staffPaymentData"
+                  @staffPaymentFormValid="staffPaymentValid=$event"
+                />
+              </v-card>
+            </div>
+            <v-card-text />
+          </section>
+        </v-col>
+
+        <v-col
+          cols="12"
+          lg="3"
+          style="position: relative"
+        >
+          <aside>
+            <affix
+              relative-element-selector="#court-order-article"
+              :offset="{ top: 120, bottom: 40 }"
+            >
+              <SbcFeeSummary
+                :filingData="filingData"
+                :payURL="getPayApiUrl"
+                @total-fee="totalFee=$event"
+              />
+              <v-card-actions class="w-full max-w-3xl mx-auto">
+                <div class="form__btns w-full">
```

Comment:
These buttons are still not displaying full-width as per the design. Please fix.